superset/views/core.py

```diff
@@ -239,8 +239,6 @@ def explore_json_data(self, cache_key: str) -> FlaskResponse:
             return json_error_response(utils.error_msg_from_exception(ex), 400)
 
     EXPLORE_JSON_METHODS = ["POST"]
-    if not is_feature_enabled("ENABLE_EXPLORE_JSON_CSRF_PROTECTION"):
```

Comment:
As this was `False` by default, shouldn't we keep the `EXPLORE_JSON_METHODS.append("GET")` ?

Reply:
Nice catch! @dpgaspar can you confirm the expected behavior?

Reply:
@geido Confirmed and added the `GET` method by default as you suggested.

Reply:
To my understanding this feature flag would allow `explore_json` to accept HTTP GETs but still imposing CSRF protection. So when `False` this endpoint would only accept HTTP POSTs and so would be automatically protected to CSRF. Shouldn't we keep the same behaviour and just allow for POST and be always protected?

Reply:
> To my understanding this feature flag would allow explore_json to accept HTTP GETs but still imposing CSRF protection.

This feature flag was `False` by default, allowing `GET`. There's no other control than excluding `GET` when this feature flag is `True`.

Reply:
missed the `not` on the feature flag check. Makes sense